devops/ci/multi_node_deploy/Jenkinsfile

```diff
@@ -0,0 +1,156 @@
+node {
```

Comment:
Rename this file to **Jenkinsfile_3Node_Deploy**.
This essentially has hard-coded values for 3 nodes. It's ok to have multiple dedicated Jenkinsfiles

Reply:
Updated